dotnet/src/Connectors/Connectors.UnitTests/Connectors.UnitTests.csproj

```diff
@@ -42,6 +42,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\SemanticKernel.Core\SemanticKernel.Core.csproj" />
+    <ProjectReference Include="..\Connectors.AssemblyAI\Connectors.AssemblyAI.csproj" />
```

Comment:
You have separate AssemblyAI.UnitTests project, you still need this line? Same for `NoWarn` and `<None Update="AssemblyAI\TestData\*">`

Reply:
Good catch